js/background.js

```diff
@@ -1,10 +1,24 @@
 
 const TIMER_NAME = "timer";
 
-var runRefresh = function(cfg) {
+function loadTabDetailsAfterLoad(tab, callback) {
+  chrome.tabs.query({windowId: tab.windowId, index: tab.index}, function (tabs) {
+    const ntab = tabs.pop();
+    if (ntab !== undefined && tab.status == "loading") {
+      const timeoutId = setTimeout(function() {
```

Comment:
So after thinking about this, we don't need to clear the timeout here. Timeout clearing should only be required if you want to implement something like racing.